lxd/main.go

```diff
@@ -818,16 +843,23 @@ in theory attack their parent container and gain more privileges than
 they otherwise would.
 
 `)
-			if askBool("Would you like to have your containers share their parent's allocation (yes/no)? ") {
+			if askBool("Would you like to have your containers share their parent's allocation (yes/no) [default=yes]? ", "yes") {
 				defaultPrivileged = 1
 			} else {
 				defaultPrivileged = 0
 			}
 		}
 
-		if askBool("Would you like LXD to be available over the network (yes/no)? ") {
-			networkAddress = askString("Address to bind LXD to (not including port): ")
-			networkPort = askInt("Port to bind LXD to (8443 recommended): ", 1, 65535)
+		if askBool("Would you like LXD to be available over the network (yes/no) [default=no]? ", "no") {
+			isIPAddress := func(s string) string {
+				if net.ParseIP(s) == nil {
+					return fmt.Sprintf("'%s' is not an IP address", s)
+				}
+				return ""
+			}
```

Comment:
Sounds good; I did it this way, but I also wouldn't be opposed to allowing (requiring?) values with []. Anyway, whichever works.